### PR TITLE
add style application layout and show items

### PR DESCRIPTION
**Pull request by Coolshreshtha**

### CODE REVIEW

File: app/javascript/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@import "~bootstrap/scss/bootstrap";
+@import url('https://fonts.googleapis.com/css?family=Merriweather:400,700');
+
+$dark: black;
+$grinnell-red: rgb(218, 41, 28);
```
Comment (spinecone) [inline app/javascript/stylesheets/application.scss:5]: Love this branding 😄

**Comment by spinecone** [comment]
Make sure bootstrap is on your resume now 💪

**Review by cvrkota — APPROVED** [review]
Looks good!

**Review by athaichi — APPROVED** [review]
Looks really good Anushka!

**Pull request merged**